metrics/default_metrics/tarantool/fibers.lua

```diff
@@ -0,0 +1,26 @@
+local fiber = require('fiber')
+local utils = require('metrics.default_metrics.tarantool.utils');
+
+local function get_fibers_metrics()
+    local fibers_info = fiber.info()
+    local fibers = 0
+    local csws = 0
+    local falloc = 0
+    local fused = 0
+
+    for id, f in pairs(fibers_info) do
+        fibers = fibers + 1
+        csws = csws + f.csw
+        falloc = falloc + f.memory.total
+        fused = fused + f.memory.used
+    end
+
+    utils.set_gauge('fiber_count', 'Amount of fibers', fibers)
+    utils.set_gauge('fiber_csw', 'Fibers csw', csws)
+    utils.set_gauge('fiber_memalloc', 'Fibers memalloc', falloc)
+    utils.set_gauge('fiber_memused', 'Fibers memused', fused)
+end
+
+return {
+    init = get_fibers_metrics,
```

Comment:
Maybe update instead of init?

We usually use init to configure a module for later usage. Think of it as a constructor.

This module doesn't have any state to configure.